### DISCUSSION TITLE
ACF text field shows all translations at once

### DISCUSSION THREAD

**Discussion by Eric-llos** [qtranslate/qtranslate-xt#1411] [Q&A]
I've cloned a page and suddenly, in the cloned one, the text fields have all the translated strings at the same time, not only the selected lang.
Is there any solution? 
<img width="1121" alt="Captura de pantalla 2024-07-03 a las 12 03 32" src="https://github.com/qtranslate/qtranslate-xt/assets/85624552/dfab816c-641a-4495-94a3-435b674d72f2">
And a video

https://github.com/qtranslate/qtranslate-xt/assets/85624552/9015cdff-00bf-4df7-b6e9-ca10cd1ac06f

Thanks! :)

**Comment by pszurgotradaway** [comment]
How do you clone posts? I also had occasional problems with this and eventually I always end up clearing the field and adding the entire field value again.

**Reply by Eric-llos** [reply]
Yoast Duplicate Post plugin can help you cloning posts. I know cleaning it is an option(the worst one if you have a lot of fields haha), but I know there's a better way to do it. Once a jobmate did something to solve it, but I can't reach him again :/
